db/schema.rb

```diff
@@ -22,6 +22,9 @@
     t.datetime "updated_at",                  null: false
     t.integer  "term_id"
     t.integer  "user_id"
+    t.integer  "upvotes_count",   default: 0
+    t.integer  "downvotes_count", default: 0
+    t.integer  "score",           default: 0
   end
```

Comment:
1. not normal to add schema without a migration, 
2. this is a method counting our upvote and downvote items ATM :)